src/postprocessing.jl

```diff
@@ -98,7 +99,8 @@ function postprocess_star_coloring!(
 
     # Process the trivial stars (if any)
     if nb_trivial_stars > 0
-        nb_unknown_hubs = nb_trivial_stars
+        occurrences .= 0
```

Comment:
```suggestion
        fill!(occurrences, 0)
```
is slightly more efficient